cheta/derived/comps.py

```diff
@@ -554,8 +554,28 @@ def get_msid_attrs(self, tstart, tstop, msid, msid_args):
 
 
 @functools.lru_cache(maxsize=1)
-def get_roll_pitch_tlm(start, stop):
```

Comment:
Looks like this is replaced by better methods, but removing it looks like an interface impact.

Reply:
I did that and it is now checked.

Reply:
This is an internal method used only in this module and it is not in the public API `__all__`.